include/cgimap/backend.hpp

```diff
@@ -24,13 +24,13 @@ struct backend {
   options() const = 0;
   // create a data selection factory from the arguments passed to cgimap.
   virtual std::shared_ptr<data_selection::factory>
-  create(const boost::program_options::variables_map &) = 0;
+  create() = 0;
```

Comment:
No longer necessary to pass `variables_map` object all over the code base since the `Options` class is a singleton and is accessible everywhere it is needed.

Reply:
I left those as is